specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -2524,13 +2528,22 @@
           "format": "int32",
           "description": "The number of replicas of the Image Version to be created per region. This property is updatable."
         },
+        "additionalReplicaSets": {
```

Comment:
let's put this on the end of properties list. so after storage acoount type

Reply:
Added